tests/ui/js/base-page-storyEditTests.js

```diff
@@ -881,44 +838,24 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             block: {
                 blockType: "text",
                 heading: "",
-                text: "",
-                simplifiedText: ""
+                text: ""
             }
         },
         "test_29": { expectedEmpty: false,
             block: {
                 blockType: "text",
                 heading: "",
-                text: "An actual text value",
-                simplifiedText: ""
+                text: "An actual text value"
             }
         },
         "test_30": { expectedEmpty: false,
             block: {
                 blockType: "text",
                 heading: "An actual heading",
-                text: "",
-                simplifiedText: ""
-            }
-        },
-        "test_31": { expectedEmpty: false,
```

Comment:
It might be useful to rename sequential numbers in keys with short meaningful test case purpose to: 1. provide a little explanation about the test case; 2. reduce the tediousness of adjusting the numbers when one or more test cases are removed.